proto/babylon/epoching/v1/params.proto

```diff
@@ -12,4 +12,25 @@ message Params {
   // epoch_interval is the number of consecutive blocks to form an epoch
   uint64 epoch_interval = 1
       [ (gogoproto.moretags) = "yaml:\"epoch_interval\"" ];
+  
+   // enqueue_gas_fees defines gas fees for different enqueue operations
+    EnqueueGasFees enqueue_gas_fees = 2
+        [ (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"enqueue_gas_fees\"" ];
+    
+  // minimum_amount is a minimum amount for staking message cancel_unbonding_delegation
+  uint64 min_amount = 3
+      [ (gogoproto.moretags) = "yaml:\"min_amount\"" ];
 }
+
+// EnqueueGasFees defines the gasFees for the enque message execution.
+  message EnqueueGasFees {
```

Comment:
Since Fees represent the value of gas * price, and what is actually needed here is just the raw gas, it seems a name change is necessary